src/lib/ares_init.c

```diff
@@ -1472,7 +1472,57 @@ static int init_by_resolv_conf(ares_channel channel)
     /* Catch the case when all the above checks fail (which happens when there
        is no network card or the cable is unplugged) */
     status = ARES_EFILE;
+#elif defined(__MVS__)
+
+  static struct __res_state *res = 0;
```

Comment:
Should this really be static?  What if system configs change?

Reply:
Yes it only needs to be done once.

Reply:
So the system config can't change?  Some apps detect system changes and will re-run ares_init() to pick up the changes and start using the new channel.  It would seem to me that declaring this static would mean that any apps attempting to re-run ares_init() after a system change wouldn't see it... Perhaps I'm missing something here.